<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutdoorSensors</class>
 <widget class="QWidget" name="OutdoorSensors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>180</width>
     <height>290</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>OutdoorWeather</string>
   </property>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-30°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+50°C</string>
    </property>
   </widget>
   <widget class="QSlider" name="sensorTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="check">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">ID 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="statusLED" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="sensorHum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>30</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelRH">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%rH</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxRH">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;100&lt;br/&gt;%rH&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinRH">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>5</y>
     <width>180</width>
     <height>290</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>UID</string>
   </property>
   <widget class="QLabel" name="labelMin_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-30°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+50°C</string>
    </property>
   </widget>
   <widget class="QSlider" name="sensorTemp_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">ID 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sensorHum_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>30</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelRH_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%rH</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxRH_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;100&lt;br/&gt;%rH&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinRH_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>5</y>
     <width>180</width>
     <height>290</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelMin_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-30°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+50°C</string>
    </property>
   </widget>
   <widget class="QSlider" name="sensorTemp_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">ID 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sensorHum_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>30</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelRH_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%rH</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxRH_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinRH_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>5</y>
     <width>180</width>
     <height>290</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="labelMin_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-30°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>+50°C</string>
    </property>
   </widget>
   <widget class="QSlider" name="sensorTemp_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>30</width>
      <height>250</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>manual</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">ID 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sensorHum_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>30</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelRH_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%rH</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxRH_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinRH_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
